admin-dev/themes/default/js/bundle/product/catalog.js

```diff
@@ -371,70 +372,70 @@ function unitProductAction(element, action) {
 }
 
 function showBulkProductEdition(show) {
-	// Paginator does not have a next page link : we are on the last page!
-	if ($('a#pagination_next_url[href]').length === 0) {
-		$('#bulk_edition_save_next').prop('disabled', true).removeClass('btn-primary');
-		$('#bulk_edition_save_keep').attr('type', 'submit').addClass('btn-primary');
-	}
-	if (show) {
-		$('#bulk_edition_toolbar').show();
-	} else {
-		$('#bulk_edition_toolbar').hide();
-	}
+  // Paginator does not have a next page link : we are on the last page!
+  if ($('a#pagination_next_url[href]').length === 0) {
+    $('#bulk_edition_save_next').prop('disabled', true).removeClass('btn-primary');
+    $('#bulk_edition_save_keep').attr('type', 'submit').addClass('btn-primary');
+  }
+  if (show) {
+    $('#bulk_edition_toolbar').show();
+  } else {
+    $('#bulk_edition_toolbar').hide();
+  }
 }
 
 function bulkProductEdition(element, action) {
-	var form = $('form#product_catalog_list');
-
-	switch (action) {
-		/*
-		case 'quantity_edition':
-			showBulkProductEdition(true);
-			$('input#bulk_action_select_all, input:checkbox[name="bulk_action_selected_products[]"]', form).prop('disabled', true);
-
-			i = 1;
-			$('td.product-sav-quantity', form).each(function() {
-				$quantity = $(this).attr('productquantityvalue');
-				$product_id = $(this).closest('tr[productid]').attr('productid');
-				$input = $('<input>').attr('type', 'text').attr('name', 'bulk_action_edit_quantity['+$product_id+']')
-					.attr('tabindex', i++)
-					.attr('onkeydown', 'if (event.keyCode == 13) return bulkProductAction(this, "edition_next"); if (event.keyCode == 27) return bulkProductEdition(this, "cancel");')
-					.val($quantity);
-				$(this).html($input);
-
-			});
-			$('#bulk_edition_toolbar input:submit').attr('tabindex', i++);
-			$('#bulk_edition_toolbar input:button').attr('tabindex', i++);
-			$('#bulk_edition_toolbar input:submit').attr('editionaction', action);
-
-			$('td.product-sav-quantity input', form).first().focus();
-			break;
-		*/
-		case 'sort':
-			showBulkProductEdition(true);
-			$('input#bulk_action_select_all, input:checkbox[name="bulk_action_selected_products[]"]', form).prop('disabled', true);
-			$('#bulk_edition_toolbar input:submit').attr('editionaction', action);
-			break;
-		case 'cancel':
-			// quantity inputs
-			$('td.product-sav-quantity', form).each(function() {
-				$(this).html($(this).attr('productquantityvalue'));
-			});
-
-			$('#bulk_edition_toolbar input:submit').removeAttr('editionaction');
-			showBulkProductEdition(false);
-			$('input#bulk_action_select_all, input:checkbox[name="bulk_action_selected_products[]"]', form).prop('disabled', false);
-			break;
-	}
+  var form = $('form#product_catalog_list');
+
+  switch (action) {
+    /*
```

Comment:
is commented code a right thing here?

Reply:
Well, it's as on the `1.7.5.x` branch: https://github.com/PrestaShop/PrestaShop/blob/7a079d7ade205ea5b9380b5f4c4ab04b848f989b/admin-dev/themes/default/js/bundle/product/catalog.js#L392

Reply:
It's commented too on `develop`
https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/themes/default/js/bundle/product/catalog.js#L391
This diff is strange 🤔 and I'm not confident with a strange diff for such a critical PR ...

Reply:
No need to worry about, you can see on the diff that the indentation of the whole file has been updated, that's why you get changes.
And we're not even talking about conflicts on this case.